uberpoet/commandlineutil.py

```diff
@@ -161,7 +161,7 @@ def make_custom_buckconfig_local(buckconfig_path):
         config.write(buckconfig)
 
 
-def count_loc(code_path):
+def count_loc(code_path, language='Swift'):
```

Comment:
Instead of making it a string, could you use a static variable in class to approach something like an enum so its less stringly typed?  This is how it's usually done in python:

```
class Foo(object):
    CONST_NAME = "Name"
```

and if you look around you'll see it in use in this project

Reply:
yeap will do that.

Reply:
done